src/App.tsx

```diff
@@ -1,26 +1,44 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42)
+const items: string[] = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [amountOfItems, setAmountOfItems] = useState(5);
+  const [currentPage, setCurrentPage] = useState(0);
```

Comment:
The currentPage state is initialized to 0, but it should be 1 as per the requirements. It says 'currentPage' (optional, defaults to 1).